ui/components/institutional/custody-confirm-link-modal/custody-confirm-link-modal.js

```diff
@@ -72,8 +72,8 @@ const CustodyConfirmLink = ({ hideModal }) => {
       category: MetaMetricsEventCategory.MMI,
       event: MetaMetricsEventName.DeeplinkClicked,
     });
-    dispatch(mmiActions.setWaitForConfirmDeepLinkDialog(false));
-    dispatch(hideModal());
+    mmiActions.setWaitForConfirmDeepLinkDialog(false);
+    hideModal();
```

Comment:
is dispatch not required for `hideModal()` ?

Reply:
Doesn't need it, as its being referenced by the parent here: https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/modals/modal.js#L401